packages/MHDDoS/MHDDoS.install

```diff
@@ -0,0 +1,13 @@
+post_install() {
+  set -e
+  cd /usr/share/MHDDoS
+  python -m venv venv
+  source venv/bin/activate &&
+    pip install --isolated --root="/usr/share/MHDDoS" --prefix='venv' \
```

Comment:
looking at https://github.com/MatrixTM/MHDDoS/blob/main/requirements.txt this package doesn't seem to require a virtual environment and a jointscript, all dependencies are in AL or BA

Reply:
ayy got it  

but we need this git+https://github.com/MHProDev/PyRoxy.git
i think its not avalable in BA or AR

Reply:
You're right, tihs one is not, in this case it need to be packaged with this template https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools, it should be pretty straightforward.